app/src/main/java/com/duckduckgo/app/browser/WebViewRequestInterceptor.kt

```diff
@@ -92,10 +94,20 @@ class WebViewRequestInterceptor(
         request: WebResourceRequest,
         webView: WebView,
         documentUri: Uri?,
+        maliciousSiteProtectionWebViewIntegration: MaliciousSiteBlockerWebViewIntegration,
         webViewClientListener: WebViewClientListener?,
     ): WebResourceResponse? {
         val url: Uri? = request.url
 
+        val confirmationCallback: (isMalicious: Boolean) -> Unit = {
```

Comment:
I assume this will have the same implementation as in `app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt` L131.

Have you considered any options to don't have them duplicated?

Reply:
I'm planning to explore it in an upcoming PR, as part of https://app.asana.com/0/72649045549333/1207151848931034